notebooks/en/rag_with_unstructured_data.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>I think it'd also be helpful to briefly explain what the other configs (ProcessorConfig, ReadConfig, SimpleLocalConfig) do.</p><p> </p><p>"...offers better OCR models for image-based documents than a local installation of Unstructured."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>